### DISCUSSION TITLE
Is there a way to get Frigate to report *where* motion is happening?

### DISCUSSION THREAD

**Discussion by DeeBeeKay** [blakeblackshear/frigate#13939] [Ask A Question] [answered]
Hello nice Frigates,

  When in Motion Tuning mode, Frigate displays lovely bounding boxes around the moving thingies. I would like to be able to get at that location data, preferably via MQTT because I sort of understand MQTT a bit.

  Is that a thing? I can't find a setting for it anywhere.

  thanks

**Comment by NickM-27** [comment]
No, this information is not available. The data would be too verbose and require too many MQTT updates across all cameras to make sense to publish.

**Comment by DeeBeeKay** [comment]
How about the 4 corners of the bounding box? The final detected one.

**Reply by NickM-27** [reply]
That's what I mean. Sending the multiple boxes that are detected, for every frame for every camera would be hundreds of messages per second or more for larger setups.

**Reply by DeeBeeKay** [reply]
I'm struggling to see how that can be true. I think perhaps we're talking about different things. Let me take another run at this.

Frigate does motion detection. It decides there's movement, and if there is, does ai detection.

If positive, then it returns, via MQTT, a number of messages, one of which is an image with a bounding box drawn around the detected item, which I use to pop up on my GUI. 

All I'm asking for is the four corners of the bounding box. 4 numbers, essentially. I'm not seeing how four numbers is vastly more data when I'm already receiving an actual image.

The only other way I can think to achieve a similar thing is to draw a grid of zones on one image, and detect based on which ones are triggering. That seems to be stupid since its essentially duplicating work.

**Reply by FabianGabor** [reply, marked as answer]
Does this from the `frigate/events` topic help?

```
{
    "before": {
        "id": "1727255287.241184-9rsvfc",
        "camera": "...",
        "frame_time": 1727255508.90988,
        "snapshot": {
            "frame_time": 1727255506.404103,
            "box": [
                1038,
                59,
                1438,
                215
            ],
            "area": 62400,
            "region": [
                967,
                0,
                1507,
                540
            ],
            "score": 0.8984375,
            "attributes": []
        },
        "label": "car",
        "box": [
            1038,
            56,
            1444,
            214
        ],
        "area": 64148,
        "ratio": 2.569620253164557,
        "region": [
            970,
            0,
            1510,
            540
        ]
    }
}
```

**Reply by blakeblackshear** [reply]
The confusion here was just terminology. You started by asking about the **motion** boxes in the motion tuner. Of which there could be many on every single frame from every camera. That would be a lot more than is sent currently.

If all you want is the bounding box for the detected object when a snapshot is provided, that's already available as noted above.

**Reply by DeeBeeKay** [reply]
Thank you @blakeblackshear. I am an Old, and as such am learning many new words.
